.github/workflows/release.yaml

```diff
@@ -226,3 +216,30 @@ jobs:
         if: success() && steps.release-branch-exists.outputs.result == 'false'
         run: |
           ./radius/.github/scripts/release-create-tag-and-branch.sh bicep-types-aws ${{ steps.get-version.outputs.release-version }} ${{ steps.get-version.outputs.release-branch-name }}
+      - name: Log in to Azure
+        uses: azure/login@v2
+        with:
+          client-id: ${{ secrets.DE_CONTAINER_AZURE_CLIENT_ID }}
+          tenant-id: ${{ secrets.DE_CONTAINER_AZURE_TENANT_ID }}
+          subscription-id: ${{ secrets.DE_CONTAINER_AZURE_SUBSCRIPTION_ID }}
+      - name: Set up Buildx
+        uses: docker/setup-buildx-action@v3
+      - name: Log in to ACR
+        if: success() && steps.release-branch-exists.outputs.result == 'false'
+        shell: bash
+        run: |
+          az acr login --name radiusdeploymentengine
+      - name: Login to GitHub Container Registry
+        if: success() && steps.release-branch-exists.outputs.result == 'false'
+        uses: docker/login-action@v3
+        with:
+          registry: ghcr.io
+          username: rad-ci-bot
+          password: ${{ secrets.GH_RAD_CI_BOT_PAT }}
+      - name: Copy Deployment Engine images from ACR to GHCR
+        if: success() && steps.release-branch-exists.outputs.result == 'false'
+        run: |
+          TAG=${{ steps.get-version.outputs.release-channel }}
+          docker buildx imagetools create \
```

Comment:
What does `buildx imagetools create` exactly do? The step title says `Copy DE images from ACR to GHCR` which sounds kind of like an easy move of images.

Reply:
Since these images are multi-arch, we can't use plain `docker pull` and `docker push`. I found this command online, it seems like exactly what we need here

https://stackoverflow.com/questions/68567983/how-to-copy-multi-arch-docker-images-to-a-different-container-registry
https://docs.docker.com/reference/cli/docker/buildx/imagetools/create/